test/json-test.js

```diff
@@ -21,7 +21,9 @@ buster.testCase('JSON API', {
         levelup(encDown(leveldown(location), {
           valueEncoding: {
             encode: JSON.stringify,
-            decode: JSON.parse
+            decode: JSON.parse,
+            type: "json",
+            buffer: false
```

Comment:
Does `{ valueEncoding: 'json' }` work?

Reply:
It _should_ and also DRY to use https://github.com/Level/codec/blob/master/lib/encodings.js#L16-L21

Reply:
yup. I will replace the commit

Reply:
ah, I think I missed those lib exports from the codec typings.